src/product-card.css

```
@@ -0,0 +1,96 @@
+.product-card {
+  width: 166px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+}
+
+.product-card_icon {
+  display: flex;
+  justify-content: center;
```

img is inline element by default, so `margin: 0 auto 40px;` don't work. Just make it block element. Flex redundant here because img have not child element

```suggestion
  display: block;
```